sqlmesh/core/renderer.py

```diff
@@ -90,12 +90,22 @@ def _render(
             The rendered expressions.
         """
 
-        cache_key = self._cache_key(start, end, execution_time, runtime_stage)
-        start_dt, end_dt, execution_dt, runtime_stage = cache_key
-        if cache_key not in self._cache:
+        cache_key = self._cache_key(start, end, execution_time)
+        start_dt, end_dt, execution_dt = cache_key
+        if runtime_stage != RuntimeStage.LOADING or cache_key not in self._cache:
             expressions = [self._expression]
 
             render_kwargs = {
+                "resolve_tables": lambda e, **kwargs: self._resolve_tables(
```

Comment:
what's this key? Should there be a better name?

Reply:
it's an expression, i can update it